microsoft-edge/webview2/howto/chromium-devtools-protocol.md

```diff
@@ -0,0 +1,190 @@
+---
+description: Learn how to use the Chrome DevTools Protocol in your WebView2 apps using the Microsoft Edge WebView2 Chromium DevTools Protocol NuGet package
+title: Use Chrome DevTools Protocol in WebView2
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 03/09/2021
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+keywords: IWebView2, IWebView2WebView, webview2, webview, edge, ICoreWebView2, ICoreWebView2Controller, Chrome DevTools Protocol
+---
+# Use Chromium DevTools Protocol in WebView2  
+
+The [Chromium DevTools Protocol][GitHubChromedevtoolsDevtoolsProtocol] provides the Chrome Debugger Extension API.  The Chrome Debugger Extension API helps you instrument, inspect, debug, and profile Chromium-based browsers.  The Chromium DevTools Protocol is the foundation for the Microsoft Edge \(Chromium\) DevTools.  The WebView team recommends you use the Chromium DevTools Protocol for features that aren't implemented in the WebView2 platform.  For more information about the Chromium DevTools Protocol functionality, navigate to Chromium DevTools Protocol.  
+
+To use the Chromium DevTools Protocol API in WebView2, use either of the following actions.  
+
+*   Install and use the [Microsoft.Web.WebView2.DevToolsProtocolExtension][NugettestIntPackagesMicrosoftWebWebView2DevToolsprotocolextension] NuGet package \(.NET\).  
+*   Run the one of the following methods.  
+    *   [CallDevToolsProtocolAsync][DotnetApiMicrosoftWebWebview2CoreCorewebview2CalldevtoolsprotocolmethodasyncViewWebview2Dotnet1077444MicrosoftWebWebView2CoreCorewebview2CalldevtoolsprotocolmethodsyncSystemStringSystemString] \(.NET\)  
+    *   [CallDevToolsProtocolMethod][Webview2ReferenceWin32Icorewebview2ViewWebview21077444Calldevtoolsprotocolmethod] \(Win32 C/C++\)  
+    
+> [!NOTE]
+> The Microsoft Edge WebView2 team does not maintain or support the Chromium DevTools Protocol.  The Chromium DevTools Protocol is maintained by the open source Chromium project.  
+> 
+> To send your suggestions for future WebView2 platform features, navigate to [WebView Feedback][GithubMicrosoftedgeWebview2feedback] and submit an issue.  
+
+## Use DevToolsProtocolHelper  
+
+[Microsoft.Web.WebView2.DevToolsProtocolExtension][NugettestIntPackagesMicrosoftWebWebView2DevToolsprotocolextension] is a NuGet package created by the WebView2 team that allows you \(a developer\) to easily access Chromium DevTools Protocol features.  The following examples describe how to use the geolocation functionality in Chromium DevTools Protocol in your WebView2 control.  You may follow a similar pattern to use other Chromium DevTools Protocol features.  
+
+## Step 1: Create a webpage to find your geolocation  
+
+To create an `HTML file` to find your geolocation, complete following the actions.  
+
+1.  Open Visual Studio Code \(or an IDE of your choice\).  
+1.  Create a new `.html` file > copy and paste the following code snippet in your new `.html` file.  
```

Comment:
What does `>` mean here?

Reply:
Updated.  

The greater-than \(`>`\) character is used to infer multiple highlights and choices in a UI chain.